### PR TITLE
fix missing permission for recreate topic

### PR DESCRIPTION
**Pull request by a1tair6**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
fix missing permission for recreate topic in **Topics** UI
missing permission [Action.MESSAGES_READ, Action.CREATE, Action.DELETE]
I just added code that Topic.tsx file

**Is there anything you'd like reviewers to focus on?**
No

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [X] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx
Comment (Haarolean) [inline kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx:75]: recreating a topic requires VIEW, CREATE and DELETE for topic, messages_read is unrelated at all

**Review by Haarolean — CHANGES_REQUESTED** [review]

File: kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx
Comment (a1tair6) [reply kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx]: @Haarolean OK. I replaced Action.MESSAGES_READ to Action.VIEW.

**Review by Haarolean — APPROVED** [review]

File: kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx:80]: shouldn't this component be converted to `ActionDropdownItem` , cause `DropdownItem` component does not have that `props`.

**Review by Mgrdich — CHANGES_REQUESTED** [review]

File: kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx
Comment (a1tair6) [reply kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx:80]: oh..i got it
and then i replaced DropdownItem to ActionDropdownItem. 
and i add disabled component.
Can i upload it?

스크린샷 2023-03-10 오전 8 52 48

Comment (a1tair6) [reply kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx]: @Mgrdich can i review it?

File: kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx:55]: do we have to disable it like this cause the old one did not have this disable prop, but the Permission parts are correctly done.

Comment (a1tair6) [reply kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx:55]: I thought it was a permission bug like the one in this picture.
image

Comment (a1tair6) [reply kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx:55]: @Mgrdich Is there anything else that needs to be modified?

**Review by Mgrdich — APPROVED** [review]

**Comment by Haarolean** [comment]
@a1tair6 please merge in the master branch

**Comment by a1tair6** [comment]
> @a1tair6 please merge in the master branch

I was trying to merge, but it got blocked due to the 'status/pending' label.

**Comment by a1tair6** [comment]
@Haarolean I was trying to merge, but it got blocked due to the 'status/pending' label.

**Comment by Haarolean** [comment]
> @Haarolean I was trying to merge, but it got blocked due to the 'status/pending' label.

You can't merge the branch *into* master, what I've been asking is to merge the master branch into your branch since
1) We can't merge an outdated branch
2) I can't merge it myself since you left no push access to your fork for maintainers

**Comment by Haarolean** [comment]
Thanks for the contribution!

**Pull request merged**
